pkg/occlient/occlient.go

```diff
@@ -1565,6 +1533,18 @@ func (c *Client) DeletePVC(name string) error {
 	return c.kubeClient.CoreV1().PersistentVolumeClaims(c.namespace).Delete(name, nil)
 }
 
+// DeleteBuildConfig deletes the given BuildConfig by name using CommonObjectMeta..
+func (c *Client) DeleteBuildConfig(commonObjectMeta metav1.ObjectMeta) error {
```

Comment:
Unit test for this

Reply:
This function is just calling Delete nothing more, I don't think that we need a unit test for this

Reply:
Just to be sure labels returned from `util.ConvertLabelsSelector()` are not modified before they are passed to the Delete call.